samcli/local/docker/lambda_image.py

```diff
@@ -308,11 +310,24 @@ def _remove_rapid_images(self, repo: str) -> None:
         try:
             for image in self.docker_client.images.list(name=repo):
                 for tag in image.tags:
-                    if ContainerManager.is_rapid_image(tag):
+                    if self.is_rapid_image(tag):
                         try:
                             self.docker_client.images.remove(image.id)
                         except docker.errors.APIError as ex:
                             LOG.warning("Failed to remove rapid image with ID: %s", image.id, exc_info=ex)
                         break
         except docker.errors.APIError as ex:
             LOG.warning("Failed getting images from repo %s", repo, exc_info=ex)
+
+    @staticmethod
+    def is_rapid_image(image_name: str) -> bool:
```

Comment:
had to move this out of ContainerManager and into here because the linter complained that there was a cyclical import if I had the constant on this file and this method in ContainerManager (since container manager used the constant but this LambdaImage used the method). The options were moving the constant to ContainerManager or moving the method here, so I thought the latter was better. I think the method might make more sense here anyway.